packages/runner/injection/multi-domain.js

```diff
@@ -40,4 +16,19 @@ const cyBridgeFrame = window.parent.frames[2]
 
 const Cypress = cyBridgeFrame.Cypress
 
+if (!Cypress) {
+  throw new Error('Something went terribly wrong and we cannot proceed. We expected to find the global \
+    Cypress in the spec bridge window but it is missing!. This should never happen and likely is a bug. Please open an issue!')
```

Comment:
Looks like this is the case in the main injection as well. Probably best to not change it or fix it in both places
```suggestion
    Cypress in the spec bridge window but it is missing! This should never happen and likely is a bug. Please open an issue!')
```

Reply:
I decided to remove the exclamation points. [49d09d1](https://github.com/cypress-io/cypress/pull/20209/commits/49d09d15f7e4262b487eefd07b5992e6971dece1)